policy/modules/system/systemd.te

```diff
@@ -1547,12 +1572,17 @@ systemd_dbus_chat_logind(systemd_user_runtime_dir_t)
 seutil_read_file_contexts(systemd_user_runtime_dir_t)
 seutil_libselinux_linked(systemd_user_runtime_dir_t)
 
+userdom_manage_user_tmp_dirs(systemd_user_runtime_dir_t)
+userdom_manage_user_tmp_files(systemd_user_runtime_dir_t)
+userdom_manage_user_tmp_pipes(systemd_user_runtime_dir_t)
+
 userdom_search_user_runtime_root(systemd_user_runtime_dir_t)
 userdom_user_runtime_root_filetrans_user_runtime(systemd_user_runtime_dir_t, dir)
 userdom_manage_user_runtime_dirs(systemd_user_runtime_dir_t)
 userdom_mounton_user_runtime_dirs(systemd_user_runtime_dir_t)
 userdom_relabelto_user_runtime_dirs(systemd_user_runtime_dir_t)
 
 optional_policy(`
-    dbus_system_bus_client(systemd_user_runtime_dir_t)
+	dbus_manage_session_dbus_runtime_sock_files(systemd_user_runtime_dir_t)
```

Comment:
I dont think systemd_user_runtime_dir_t has any business with that sock file. All it should be doing is create that /run/user/* dir and mount a tmpfs on it AFAIK

Reply:
I added this because of this denial:
```
time->Fri Mar 12 16:33:38 2021
type=PROCTITLE msg=audit(1615584818.382:3908): proctitle=2F6C69622F73797374656D642F73797374656D642D757365722D72756E74696D652D64697200
73746F700031303030
type=PATH msg=audit(1615584818.382:3908): item=1 name="bus" inode=15 dev=00:26 mode=0140666 ouid=1000 ogid=1000 rdev=00:00 obj=staff_
u:object_r:session_dbusd_runtime_t:s0 nametype=DELETE cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=PATH msg=audit(1615584818.382:3908): item=0 name="/" inode=1 dev=00:26 mode=040700 ouid=1000 ogid=1000 rdev=00:00 obj=staff_u:ob
ject_r:user_runtime_t:s0 nametype=PARENT cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=CWD msg=audit(1615584818.382:3908): cwd="/"
type=SYSCALL msg=audit(1615584818.382:3908): arch=c000003e syscall=263 success=no exit=-13 a0=4 a1=25323de3 a2=0 a3=76 items=2 ppid=1
 pid=146579 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="systemd-user-ru" 
exe="/lib/systemd/systemd-user-runtime-dir" subj=system_u:system_r:systemd_user_runtime_dir_t:s0 key=(null)
type=AVC msg=audit(1615584818.382:3908): avc:  denied  { unlink } for  pid=146579 comm="systemd-user-ru" name="bus" dev="tmpfs" ino=1
5 scontext=system_u:system_r:systemd_user_runtime_dir_t:s0 tcontext=staff_u:object_r:session_dbusd_runtime_t:s0 tclass=sock_file perm
issive=0
```
But I think this can probably be reduced to only allowing unlinking of that sock file. What do you think?